pkg/kubelet/winstats/perfcounters.go

```diff
@@ -0,0 +1,104 @@
+// +build windows
```

Comment:
Maybe merge this and `helpers.go` into a single file?

Reply:
Done.